src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetCultureCommand.cs

```diff
@@ -1,23 +1,123 @@
+    /// Returns:
+    ///     - the thread's current culture
+    ///     - culture by name
+    ///     - list of all supported cultures.
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, "Culture", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113312")]
+    [Cmdlet(VerbsCommon.Get, "Culture", DefaultParameterSetName = CurrentCultureParameterSet, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113312")]
     [OutputType(typeof(System.Globalization.CultureInfo))]
     public sealed class GetCultureCommand : PSCmdlet
     {
+        private const string CurrentCultureParameterSet = "CurrentCulture";
+        private const string NameParameterSet = "Name";
+        private const string ListAvailableParameterSet = "ListAvailable";
+
+        /// <summary>
+        /// Gets or sets culture names for which CultureInfo values are returned.
+        /// </summary>
+        [Parameter(ParameterSetName = NameParameterSet, Position = 0, ValueFromPipeline = true)]
+        [ValidateSet(typeof(ValidateCultureNamesGenerator))]
+        [ValidateNotNull]
+        public string[] Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets a switch to return current culture with user overrides (by default).
+        /// With the switch on, we return current culture without user overrides.
+        /// </summary>
+        [Parameter(ParameterSetName = CurrentCultureParameterSet)]
+        [Parameter(ParameterSetName = NameParameterSet)]
+        public SwitchParameter NoUserOverrides { get; set; }
+
+        /// <summary>
+        /// Gets or sets a switch to list all available cultures.
+        /// </summary>
+        [Parameter(ParameterSetName = ListAvailableParameterSet)]
+        public SwitchParameter ListAvailable { get; set; }
+
         /// <summary>
-        /// Output the current Culture info object.
+        /// Output:
+        ///     - the thread's current culture
+        ///     - culture by name
+        ///     - list of all supported cultures.
         /// </summary>
-        protected override void BeginProcessing()
+        protected override void ProcessRecord()
+        {
+            CultureInfo ci;
+
+            switch (ParameterSetName)
+            {
+                case CurrentCultureParameterSet:
+                    if (NoUserOverrides)
+                    {
+                        ci = CultureInfo.GetCultureInfo(Host.CurrentCulture.LCID);
+                    }
+                    else
+                    {
+                        ci = Host.CurrentCulture;
+                    }
+
+                    WriteObject(ci);
+
+                    break;
+                case NameParameterSet:
+                    try
+                    {
+                        foreach (var cultureName in Name)
+                        {
+                            if (NoUserOverrides && string.Equals(cultureName, Host.CurrentCulture.Name, StringComparison.CurrentCultureIgnoreCase))
```

Comment:
This should be `!NoUserOverrides` and the 2nd `CultureInfo` constructor argument below should be `true` (or omitted), but you could just use `Host.CurrentCulture`, which also reflects the overrides.

Reply:
Fixed.